v4/weaviate/connection/rest.go

```diff
@@ -77,7 +132,7 @@ func (con *Connection) createRequest(ctx context.Context, path string,
 		return nil, err
 	}
 	con.addHeaderToRequest(request)
-	//con.addURLParametersToRequest(request, urlParameters)
+	// con.addURLParametersToRequest(request, urlParameters)
```

Comment:
can this be removed?

Reply:
That was there before, but I'll delete it. Nobody seems to miss it and should be easy to readd

Reply:
Ah , ok 👍